pkg/util/util.go

```diff
@@ -59,13 +57,13 @@ func ParseEndpoint(endpoint string) (string, string, error) {
 		return "", "", fmt.Errorf("could not parse endpoint: %v", err)
 	}
 
-	addr := path.Join(u.Host, filepath.FromSlash(u.Path))
```

Comment:
Tests are failing, this code is pretty confusing.

https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/pkg/util/util_test.go

https://golang.org/pkg/net/url/#URL

`tcp:///127.0.0.1` should mean "listen on path /127.0.0.1"  but we are twisting it to be equivalent to `tcp://127.0.0.1` which should mean "listen on host 127.0.0.1". This seems wrong but I don't care to change it now.

For unix scheme, https://github.com/grpc/grpc/blob/master/doc/naming.md#name-syntax suggests we should support `unix:path` and `unix://absolute-path`. golang url.Parse will parse the first one perfectly but the first segment of the `absolute-path` in the second one will be parsed as the host.

In interest of not breaking anything I will make this change in a future PR so that tests pass and things work on Windows...